dnsapi/dns_ddnss.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env sh
+
+#Created by RaidenII, to use DuckDNS's API to add/remove text records
+#06/27/201
+#modified by helbgd @ 03/13/2018 to support ddnss.de
+#Please note: the Wildcard Feature must be turned on for the Host record
+#and the checkbox for TXT needs to be enabled
+
+# Pass credentials before "acme.sh --issue --dns dns_ddnss ..."
+# --
+# export DDNSS_Token="aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
+# --
+#
+
```

Comment:
another empty line deleted

Reply:
@helbgd Just wanted to let you know ,that I modified your script very slightly and made a pull request that just got merged. Thank you very much for your great work (I hope it is ok for you that I took your script and modified it. Of course I left all the credits in there...)